multiqc/modules/dragen/utils.py

```diff
@@ -141,3 +141,182 @@ def make_headers(parsed_metric_ids, metrics):
 
 def exist_and_number(data, *metrics):
     return all(isinstance(data.get(m, None), int) or isinstance(data.get(m, None), float) for m in metrics)
+
+
+def check_duplicate_samples(sample_names, logger, module):
+    """Check samples for duplicate names. Warn about found ones."""
+    message = ""
+    line1 = "\n  {} was built from the following samples:"
+    line2 = "\n    {} in {}"
+    for clean_sample_name in sample_names:
+        if len(sample_names[clean_sample_name]) > 1:
+            message += line1.format(clean_sample_name)
+            for file_handler in sample_names[clean_sample_name]:
+                message += line2.format(file_handler["fn"], file_handler["root"])
+    if message:
+        message = "\n\nDuplicate sample names were found. The last one overwrites previous data." + message
+        logger.warning(message)
```

Comment:
Please stick to the standard way of handling this for MultiQC, documented here: https://multiqc.info/docs/development/modules/#identical-sample-names

There are many reasons why duplicate sample names are found when running MultiQC, most are harmless. This warning log message is much too noisy.

Reply:
Switched to `logger.debug` instead. Resolved